docs/src/modules/java/pages/model-provider-details.adoc

```diff
@@ -258,6 +262,27 @@ See link:_attachments/api/akka/javasdk/agent/ModelProvider.OpenAi.html[`ModelPro
 == Reference configurations
 The following is a list of the various reference configurations for each of the AI models
 
+Note that the following reference configurations are the default values, and you would typically only define the properties that you want to override, such as:
+
+[source,hocon,indent=0]
+----
+akka.javasdk.agent.openai {
+  model-name = "gpt-4o-mini"
+}
+----
+
+You may also have to use a fallback to the reference configuration if you use a different configuration section:
```

Comment:
This is the case that was reported in the issue.

I considered making all properties optional (if hasPath), but that is not great with for api-key, which has env variable default from reference conf, and makes things more complicated for something that shouldn't be that common.